DirectEmployers/MyJobs
# Misc Fixes from Router Refactor

## What's this PR do?

Cleans up some promise calls and removes unused code.

## Screenshots (if appropriate)

## Where should the reviewer start?

## How should this be manually tested?

Check to see that the apps in question still work. 
(1) Reporting
(2) Manage Users
(3) Nonuser Outreach

## Any background context you want to provide?
Sometimes I write code bad
